R/report.htest.R

```diff
@@ -84,14 +93,13 @@ report_effectsize.htest <- function(x, ...) {
       table <- table[c(estimate)]
     }
 
-    # TODO: Chi square test
+    # TO DO: Chi-squared test -------------
```

Comment:
Is this also recognized by the _Todor_ addin?

Reply:
Oops, that was unintended. I have reverted that change. 

I had no idea about this addin. Thanks for the tip!